manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
this file doesn't seem to be required

Reply:
Thought it would be easier for future people to create migrations :D But I can remove it